### PR TITLE
Release `v9.2.0`

### PR DESCRIPTION
**Pull request by Mati365**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Changelog for `v9.2.0`.

### CODE REVIEW

**Comment by pomek** [comment]
We need a green light from @Witoso to push this release forward.

File: CHANGELOG.md
Comment (Witoso) [inline CHANGELOG.md:8]: Please add that it's Experimental, someone could have skipped it in the previous release.

File: CHANGELOG.md
Comment (Witoso) [inline CHANGELOG.md:12]: Please add that it's Experimental, someone could have skipped it in the previous release.

**Review by Witoso — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (Mati365) [reply CHANGELOG.md]: Done

**Review by Witoso — DISMISSED** [review]

**Review by pomek — DISMISSED** [review]

File: CHANGELOG.md
Comment (pomek) [inline CHANGELOG.md:12]: I am unsure if it's a fix or _Other change_ as it looks like "aligning to a dependency API" instead of "fixing something because we made a mistake".

**Comment by pomek** [comment]
Once the PR is merged, please ping @psmyrek / @martnpaneq to process the publishing process.

File: CHANGELOG.md
Comment (Mati365) [reply CHANGELOG.md:12]: @pomek renamed

**Review by pomek — APPROVED** [review]

**Pull request merged**
